sw/device/silicon_creator/rom/e2e/BUILD

```diff
@@ -3920,36 +3920,46 @@ test_suite(
 opentitan_functest(
     name = "presigned_fpga_binary",
     cw310 = cw310_params(
-        bitstream = ":bitstream_sigverify_spx_prod_enabled_zero",
+        bitstream = "//hw/bitstream:rom_with_real_keys_otp_test_unlocked0",
```

Comment:
note that this test was signed with both RSA and SPHINCS+ keys. This change seems to be for RSA-only signature verification. is the RSA + SPHINCS+ case tracked somewhere?

Reply:
See #18860.

Because we don't have a secure signing solution for SPX keys (HSM or personal token), I'm nervous about signing this and then having to make a last minute change to the ROM.

Reply:
I had the self hash test on the brain - we can probably sign the `empty_test` with an SPX key and test that.

Reply:
sg, thanks!